File: Python/Координаты.py
x=int(input("Введите координату по оси x: "))
y=int(input("Введите координату по оси y: "))
if x>0 and y>0:
    print("I четверть")
elif x<0 and y>0:
    print("II четверть")
elif x<0 and y<0:
    print("III четверть")
elif x>0 and y<0:
    print("IV четверть")
else:
    print("Точка не находится в координатной четверти")

File: C++/Координаты.cpp
#include<iostream>
using namespace std;
int main () {
	int x;
	int y;
	cin>>x>>y;
	setlocale(LC_ALL, "Russian");
	if (x>0 && y>0){
		cout<<"I четверть";
}
	else if (x<0 && y>0) {
		cout<<"II четверть";
}
	else if (x<0 && y<0) {
		cout<<"III четверть";
}
	else if (x>0 && y<0) {
		cout<<"IV четверть";
}
	else {
		cout<<"Точка находится в середине";
}	
}
